src/Windows/Perspex.Win32/Interop/UnmanagedMethods.cs

```diff
@@ -751,6 +754,8 @@ internal class FileOpenDialogRCW { }
         [DllImport("comdlg32.dll")]
         public static extern int CommDlgExtendedError();
 
+        public static bool ShCoreAvailable => LoadLibrary("shcore.dll") != IntPtr.Zero;
```

Comment:
Ah yes, you're right. I mistakenly thought that it `GetDpiForMonitor` was called every time the DPI changes. It's probably not a big deal then.